kimchi/src/circuits/polynomials/keccak/circuitgates.rs

```diff
@@ -220,12 +315,13 @@ where
         let mut constraints = vec![];
 
         // LOAD WITNESS
-        let old_state = env.witness_curr_chunk(0, 100);
-        let new_block = env.witness_curr_chunk(100, 200);
-        let zeros = env.witness_curr_chunk(168, 200);
-        let xor_state = env.witness_next_chunk(0, 100);
-        let bytes = env.witness_curr_chunk(200, 400);
-        let shifts = env.witness_curr_chunk(400, 800);
+        let old_state = env.witness_curr_chunk(SPONGE_OLD_STATE_OFF, SPONGE_OLD_STATE_LEN);
```

Comment:
Do you want the length of the new state offset?

Reply:
well in this case it is equivalent (because the offset of the old state is 0), but let's do new state offset so they all have the same shape